src/Microsoft.Diagnostics.Monitoring.WebApi/ProcessInfoImpl.cs

```diff
@@ -92,7 +92,19 @@ public static async Task<IProcessInfo> FromEndpointInfoAsync(IEndpointInfo endpo
 
                     commandLine = await commandLineSource.Task;
                 }
-                catch
+                catch (ObjectDisposedException)
```

Comment:
I wasn't super confident in expanding this one, but this was my best guess based on looking at the code in the `try`.

Reply:
Let's remove ObjectDisposedException, ArgumentNullException, and ArgumentException catchers.